docs/reference/verify-assets.mdx

```diff
@@ -160,63 +219,70 @@ WARN[0000] image signature verification is disabled due to using corresponding o
 WARN[0000] image signature verification is disabled due to using corresponding option
 RUNTIME.CONTAINERNAME  PID          UID          GID          MNTNS_ID RET FL… MODE        COMM        FNAME                  TIMESTAMP
 ```
+</TabItem>
 
-### With `kubectl-gadget`
+<TabItem value="ig-daemon" label="ig daemon">
 
-Compared to `ig`, you cannot specify information with regard to verification when calling `kubectl gadget run`:
+Like `kubectl-gadget`, you cannot specify information with regard to verification when calling `gadgetctl run`:
 
 ```bash
-$ kubectl gadget run --public-keys="$(cat your-key.pub)" trace_exec
+$ gadgetctl run --public-keys="$(cat your-key.pub)" trace_exec --remote-address=$addr
 Error: unknown flag: --public-keys
-$ kubectl gadget run --verify-image=false trace_exec
+$ gadgetctl run --verify-image=false trace_exec --remote-address=$addr
 Error: unknown flag: --verify-image
 ```
 
-Instead, all these information are set once at deploy time.
+Instead, all these information are set once when starting `ig` as a daemon.
 This avoid them being tampered by any users.
 By default all image-based gadgets are verified using Inspektor Gadget public key:
 
 ```bash
-$ kubectl gadget deploy
-...
-Inspektor Gadget successfully deployed
-$ kubectl gadget run ghcr.io/your_repo/trace_exec -A
-Error: fetching gadget information: getting gadget info: rpc error: code = Unknown desc = getting gadget info: initializing and preparing operators: instantiating operator "oci": ensuring image: verifying image "ghcr.io/your_repo/trace_exec": verifying signature: invalid signature when validating ASN.1 encoded signature
-$ kubectl gadget run trace_exec -A
-K8S.NAMESP… K8S.PODNAME K8S.CONTAI… MNTNS… TIMES… PID    PPID   UID    GID    LOGIN… SESSI… RETVAL ARGS_… UPPER… ARGS_… COMM  ARGS  K8S.…
-gadget      gadge…hbh8n gadget      40265… 22867… 53386  53368  0      0      42949… 42949… 0      2      false  35     gadg… /bin… mini…
-gadget      gadge…hbh8n gadget      40265… 22867… 53387  53369  0      0      42949… 42949… 0      2      false  35     gadg… /bin… mini…
+$ IP='your_ip'
+$ PORT='your_port'
+$ ADDR=tcp://$IP:$PORT
+$ sudo ig daemon --public-keys="$(cat your-key.pub),$(cat inspektor-gadget.pub)" --host=$ADDR
```

Comment:
See above: `--host` is not the focus of this example, so IMHO it should be omitted.